mtp_api/apps/credit/management/commands/create_prisoner_credit_notices.py

```diff
@@ -37,11 +37,11 @@ def handle(self, path, prison, date=None, **options):
         try:
             prison = Prison.objects.get(pk=prison)
         except Prison.DoesNotExist:
-            raise CommandError('Prison %s does not exist' % prison)
+            raise CommandError('Prison does not exist', {'prison_nomis_id', prison})
```

Comment:
Does raising a `CommandError` exception behaves like using `logger.error()` in the sense that this other parameter ends up being in the Sentry context?

Reply:
it'd appear in a similar way in sentry, but the title of the error is just the exception class name (`CommandError` in this case) with the description being a stringified version of all arguments (`('Prison does not exist', {'prison_nomis_id', ????})` in this case)